x/claim/genesis.go

```diff
@@ -10,31 +10,21 @@ import (
 // state.
 func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState) {
 	k.SetModuleAccountBalance(ctx, genState.ModuleAccountBalance)
```

Comment:
IMHO, I think rather than setting the module account balance of the claims module at genesis, we should use the balance of by adding all claimable balances.
While this would require more computation at genesis, precalculating all the balances for genesis.json can be tedious work and can be prone to human mistakes.

Reply:
@Thunnini how about I add it as a ValidateGenesis check?

Reply:
Ok, sounds like a good idea 👍

Reply:
Done